### ISSUE TITLE
Routiu

### ISSUE THREAD

**Issue by zieglerdo** [abrensch/brouter#425]

**Comment by polyscias** [comment]
Yes, I can reproduce the problem on brouter-web and indeed strange.

I am wondering if this is a side-effect of Douglas-Peucker transfer-node elimination, see [d2aaeb2](https://github.com/abrensch/brouter/commit/d2aaeb2988b95d7be3c9601faa4e1cb6e2132ae4) and [aa7bb37](https://github.com/abrensch/brouter/commit/aa7bb37a99acf980de232fe5d6cbe70ee44a9cf2)

**Comment by polyscias** [comment]
With the trekking profile I could trigger even more strange behavior:

http://brouter.de/brouter-web/#map=19/63.37879/10.34758/standard&lonlats=10.346938,63.378979;10.34704,63.378981

That can not be due to Douglas-Peucker transfer-node elimination.

**Comment by quaelnix** [comment]
This no longer seems to be the case, do we know why?

**Comment by afischerdev** [comment]
Yes, I think so. When I played with this problem there wasn't a way around. Now the street is splitted.
The [way has changed](https://www.openstreetmap.org/way/1143725548) 6 month ago.  

But the issue with loops is still present. Please see this [forum entry](https://groups.google.com/g/osm-android-bikerouting/c/iCDokBDENM0/m/yVXZruXDDwAJ).
